README.md

```diff
@@ -135,7 +136,7 @@ NoSSLv3   | When `true`, the SSLv3 protocol is disabled. If NoSSLv2 and NoSSLv3
 NoTLSv1_0 | When `true`, the TLSv1.0 protocol is disabled
 NoTLSv1_1 | When `true`, and if available on the platform, the TLSv1.1 protocol is disabled
 NoTLSv1_2 | When `true`, and if available on the platform, the TLSv1.2 protocol is disabled
-sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more information, see [OpenSSL's documentation](https://openssl.org/docs/manmaster/apps/ciphers.html "OpenSSL's documentation")
+sslCipherSuite | The prioritized list of allowed SSL/TLS ciphers. For more information, see [OpenSSL's documentation](https://www.openssl.org/docs/manmaster/man1/ciphers.html "OpenSSL's documentation")
```

Comment:
This **WAS** a good link, and the OpenSSL team obviously did some documentation reorg. What makes you think this will be a good link tomorrow or the next day?

How about referring to something a little more generic, like issuing the command `man ciphers` on a Linux box with MAN pages installed? That way, the OpenSSL team can reorg their docs to their heart's content, and we're fine. If MAN pages are not installed, then suggest a WWW search for something like `openssl man ciphers`?

Reply:
Also, merge commits please.